src/getMovieRating.js

```diff
@@ -0,0 +1,22 @@
+import { loadReviewsForMovie } from "./movies.js";
+import { loadIMDBMovieRating } from "./omdbapi.js";
+
+export default async function getMovieRating(movieID, imdbID, reviewSource1 = loadReviewsForMovie(movieID), reviewSource2 = loadIMDBMovieRating(imdbID)) {
```

Comment:
I would strongly recommend against executing functions in the parameter list like this. Although possible, it's very unusual, which makes it unexpected and difficult to read. It becomes extra confusing when the functions are asynchronous.

I'm guessing you've done this to be order to mock the data. If that's the purpose, a simpler solution would be to pass in the _functions_, not execute them and pass in their return values.

```js
export default async function getMovieRating(movieID, imdbID, loadFromCms = loadReviewsForMovie, loadFromImdb = loadIMDBMovieRating) {
  const reviewList = await loadFromCms(movieId);
  // …
}
```